CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+
+```shell
+https://github.com/boxyhq/jackson
+```
+
+### 2. Clone the Repository
+
+Clone your forked repository to your local development environment using this command:
+
+```shell
+git clone https://github.com/your-username/jackson.git
+```
+
+### 3. Set Up
+
+Navigate to the project folder and install necessary dependencies:
+
+```shell
+cd jackson
+npm run custom-install
+```
+
+#### Add Environment Variables
+
+```shell
+npm run dev
+cp .env.example .env
+```
+
+Please update .env with your values. See the complete list of [Environment Variables](https://boxyhq.com/docs/jackson/deploy/env-variables).
+
+### 4. Build and Run
+
+Ensure the project is ready for development:
+
+```shell
+npm run build
+npm run start
+```
+
+Visit [http://localhost:5225](http://localhost:5225) in your browser. If you see a sign-in page, you're on the Admin Portal.
+
+Read the full documentation [here](https://boxyhq.com/docs/jackson/deploy/).
+
+## Contribution Guidelines
+
+
+### Create a new branch 
+
+```shell
+git checkout -b your-branch-name
+```
+#### or
+
+```shell
+
+git branch your-branch-name
+```
+### Make and commit your changes
+
+```shell
+
+git commit -m ""enter the message for changes that need to be committed"
+```
+
+### Push your changes by commiting them
+
+```shell
+git push origin your-branch-name
+```
+
+### Create a Pull Request
+
+Good pull requests - patches, improvements, new features - are a fantastic help. They should remain focused in scope and avoid containing unrelated commits.
+
+Go to the original repository on GitHub and click the "New Pull Request" button. Compare and create a pull request from your branch to the main repository. Provide a clear and concise description of your changes in the pull request.
+
+### Review and Feedback
+
+After submitting your PR, maintainers and other contributors may review your changes and provide feedback. Be ready to address any suggested improvements.
+
+### Merge
+
+Once your PR is approved, it will be merged into the main repository.
+
+Celebrate!
+
+
+### Bug Reports
+
+If you encounter issues or bugs, please report them with detailed information.
+
+### Feature Requests
+
+Submit detailed feature requests for new functionality.
+
+### Testing
+
+Thoroughly test your changes to avoid regressions and errors.
+
+### Good First Issues
+
+Start your contribution journey with our list of "good first issue" tasks, ideal for newcomers and first timers.
+
+## Development
+
+1. Fork the repository.
+2. Clone your fork to your local environment.
+3. Install dependencies.
+4. Implement changes, add features, or enhance documentation.
+5. Test your changes.
+6. Submit a pull request against the main branch.
+
+## Code Of Conduct 
+
+Please ensure that your contributions align with our [Code of Conduct](https://github.com/boxyhq/jackson/blob/6cf5639706714c727d093136372a46ac7b26ac0a/CODE_OF_CONDUCT.md#L1). Be respectful, inclusive, and considerate of others.
```

Comment:
Please point it to main branch and without the L1 anchor

Reply:
@VanshBhardwaj24 This one is the last pending issue

Reply:
@deepakprabhakara fixed this and thank you so much for your guidance as i am beginner